src/main/java/com/visenze/visearch/ImageResult.java

```diff
@@ -9,6 +10,8 @@
     private final Float score;
     private String s3Url;
     private Map<String, String> vsMetadata;
+    private List<ImageResult> alternatives;
+    private Map<String, String> tags;
```

Comment:
i am not sure whether the tags will only be String to String, some tags may have multiple values even for category. e.g. "category" : ["fashion", "shoe"]
Let me comment on original API design

Reply:
I'll use Object here for the flexibility.

Reply:
i just confirm with Todd/Yuwen, it will be String, String